test/mann_whitney.jl

```diff
@@ -53,4 +53,22 @@ end
 @testset "Issue #39" begin
 	@test abs(pvalue(ExactMannWhitneyUTest(Float32[1:10;], Float32[2:11;])) - 0.5096) <= 1e-4
 end
+
+@testset "Issue #126 pvalues above 1" begin
+	A = [1.34937, 1.75722,0.276514, 1.04546, 1.69085, 0.738085, 2.36313]
+	B = [2.62325, 1.16533, 1.1327, 0.728714]
+
+	m = ExactMannWhitneyUTest(A,B)
+	p = pvalue(m; tail = :both)
+	@test p == 1
+
+	A = [12,10,7,6,3,1]
+	B = [11,9,8,5,4,2]
+
+	m = MannWhitneyUTest(A,B)
```

Comment:
This will also be an `ExactMannWhitneyUTest`, so the changes for the p-value and type stability in the `ApproximateWhitneyUTest` are not tested.

I wanted to suggest changing this to
```julia
	m = ApproximateMannWhitneyUTest(A,B)
```
but unfortunately the p-value is already 1 for this example so it can only be used for testing type stability of `pvalue`.

Reply:
I can't find inputs to get the p-value above `1` here. But I did fix an inference issue, which is nice.